README.md

```diff
@@ -9,6 +9,7 @@
 
 Supported tags and release links
 
+* [3.2.1](https://github.com/dtzar/helm-kubectl/releases/tag/3.1.2) - helm v3.2.1, kubectl v1.18.2, alpine 3.11
```

Comment:
Fix tag URI 3.1.2 --> 3.2.1.
If you can fix the previous tag URI too that'd be appreciated ;)

Reply:
Done :)